src/main/java/org/broadinstitute/hellbender/tools/exome/HetPulldownCalculator.java

```diff
@@ -31,8 +31,12 @@
     private final File refFile;
     private final IntervalList snpIntervals;
 
+    private static final int NUMBER_OF_LOG_UPDATES = 20;
+    private static final double HET_ALLELE_FRACTION = 0.5;
+
     /** Set quality and read-depth thresholds for pulldown, interval threshold for indexing for SamLocusIterator. */
-    private static final int MIN_QUALITY = 0;
+    private static final int MIN_BASE_QUALITY = 20;
```

Comment:
Are you sure that you want these hardcoded and not an advanced parameter with a default value?